engine/sound/src/sound.cpp

```diff
@@ -281,6 +282,24 @@ namespace dmSound
         return RESULT_DEVICE_NOT_FOUND;
     }
 
+    static float GainToScale(float gain)
+    {
+    #if defined(DM_SOUND_USE_LEGACY_GAIN) && (DM_SOUND_USE_LEGACY_GAIN != 0)
+        return gain;
+    #else
+        gain = dmMath::Clamp(gain, 0.0f, 1.0f);
+        // Convert "gain" to scale so progression over the range 'feels' linear
+        // (roughly 60dB(A) range assumed; rough approximation would be simply X^4 -- if this ever is too costly)
+        const float l = 0.1f;   // linear taper-off range
+        const float a = 1e-3f;
```

Comment:
Feel free to use `0.001f` here for readability (especially as the other ones are in decimal form)